compiler/rustc_mir_build/src/build/matches/test.rs

```diff
@@ -499,7 +499,7 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
     /// However, in some cases, the test may just not be relevant to candidate.
     /// For example, suppose we are testing whether `foo.x == 22`, but in one
     /// match arm we have `Foo { x: _, ... }`... in that case, the test for
-    /// what value `x` has has no particular relevance to this candidate. In
+    /// what value `x` has no particular relevance to this candidate. In
```

Comment:
this isn't technically a spelling error, but spell checkers have been flagging doubled words for decades.
sometimes doubled words are intentional -- this doesn't appear to be one such case.

Reply:
the double word is correct here (although it makes the sentence a little awkward) - feel free to either reword the sentence of drop the change